lib/webpack_manifest/rails.rb

```diff
@@ -13,6 +20,33 @@ def configuration(&block)
         @configuration
       end
       attr_writer :configuration
+
+      def build(logger: nil)
+        logger ||= Logger.new(STDOUT).tap do |l|
```

Comment:
If the method is located here, it should not make output to stdout because here is in a library scope. Can the logger be set only if it is called in the context of rspec? 🤔